### ISSUE TITLE
Tune MAE training recipe

### ISSUE THREAD

**Issue by clane9** [MedARC-AI/fmri-fm#14]
Our current [default training recipe](https://github.com/MedARC-AI/fmri-fm/blob/b3d97ca3b5bd5ce003628236811136d56c7f4ece/src/flat_mae/config/default_pretrain.yaml) is fairly arbitrary and untuned. We should do at least some tuning (current defaults in **bold**)

- batch size: **32**, 64, 128, ...
- epochs: **100**, 200, 400, ...
- learning rate: 5e-4, **1e-3**, 2e-3, ...

Other settings that could be worth taking a look at:
- repeated sampling rate ([`oversample`](https://github.com/MedARC-AI/fmri-fm/blob/b3d97ca3b5bd5ce003628236811136d56c7f4ece/src/flat_mae/config/default_pretrain.yaml#L107))
- gradient clipping
- shuffle buffer size
